### DISCUSSION TITLE
Vulnerability Management - Pnpm

### DISCUSSION THREAD

**Discussion by ngowraj** [pnpm/pnpm#5393] [Q&A]
We recently transitioned from npm tp pnpm for performance reasons. Before the transition we were using Github's [dependabot](https://github.com/dependabot) and [Snyk](https://snyk.io/) for identifying security vulnerabilities within our open source libraries. 

After the transition, both the products fail to report any security vulnerabilities since they don't support pnpm. I was wondering if the community has any solutions or recommendations for an automated approach of identifying and patching security vulnerabilities?

**Comment by ronjouch** [comment]
You might be hitting https://github.com/dependabot/dependabot-core/issues/13177 , which I mentioned at https://github.com/orgs/pnpm/discussions/3787#discussioncomment-15488747
